sql/core/src/main/scala/org/apache/spark/sql/execution/debug/package.scala

```diff
@@ -81,29 +100,11 @@ package object debug {
      * WholeStageCodegen subtree).
      */
     def debugCodegen(): Unit = {
-      debugPrint(debugCodegenString())
+      debugPrint(codegenString(query.queryExecution.executedPlan))
     }
 
     /** Visible for testing. */
-    def debugCodegenString(): String = {
-      val plan = query.queryExecution.executedPlan
-      val codegenSubtrees = new collection.mutable.HashSet[WholeStageCodegen]()
-      plan transform {
-        case s: WholeStageCodegen =>
-          codegenSubtrees += s
-          s
-        case s => s
-      }
-      var output = s"Found ${codegenSubtrees.size} WholeStageCodegen subtrees.\n"
-      for ((s, i) <- codegenSubtrees.toSeq.zipWithIndex) {
-        output += s"== Subtree ${i + 1} / ${codegenSubtrees.size} ==\n"
-        output += s
-        output += "\nGenerated code:\n"
-        val (_, source) = s.doCodeGen()
-        output += s"${CodeFormatter.format(source)}\n"
-      }
-      output
-    }
+    def debugCodegenString(): String = codegenString(query.queryExecution.executedPlan)
```

Comment:
It is still used in `DebuggingSuite` since I didn't change that testcase.
```
val res = sqlContext.range(10).groupBy("id").count().debugCodegenString()
```